packages/frontend/web/server/htmlTemplate.ts

```diff
@@ -37,7 +37,7 @@ export const htmlTemplate = ({
     );
 
     const lowPriorityScriptTags = lowPriorityScripts.map(
-        src => `<script defer src="${src}"></script>`,
+        src => `<script async src="${src}"></script>`,
```

Comment:
Is execution order likely to be important for low priority scripts?

Reply:
I think, no, other than waiting for polyfills. This is what spawned this issue though (it was async but I moved to defer in another PR to get around the issue), because polyfill.io was defer and these were async, so these were erroring. I think this is probably best of both worlds.